docs/walkthrough.rst

```diff
@@ -0,0 +1,199 @@
+expected by the import .csv script. This has two use:
+
+* export existing key-value pairs (along other things like namespace or tags).
+  This can be useful to modify them before re-import, or to transfer annotations
+  across groups.
+* export a list of objects with their name and ID.
+
+.. figure:: images/annotation_04_export_output.png
+
+   *Exported .csv with only the object ID and name*
+..
+
+  Tip: If you have Key-Value pairs attached to your objects that you do not \
+  wish to export (to create a template like shown above), specify a not-in-use \
+  namespace (any random input will work). This does not affect tags.
+
+
+Importing annotations
+---------------------
+
+Starting from the file we exported as explained in the previous section, \
+we proceed to edit it within a spreadsheet editor, adding more columns to the csv, \
+including a row specifying the namespace to assign for each key:
+
+.. figure:: images/annotation_05_KV_to_import.png
+
+   *Result of populating the csv shown in a text editor*
+..
+
+We added several columns to annotate our dataset with Key-Value pairs \
+following the `REMBI <https://doi.org/10.1038/s41592-021-01166-8>`_ guidelines \
+(after saving the document, our ``TAB`` separator became ``,``).
+
+We proceed and start the script "*Import from CSV*".
+
+.. figure:: images/annotation_06_script_import.png
+
+   *Selection of all the datasets of a project and the Import from CSV script.*
+..
+
+There are many parameters here that could help you fine tune the way to import annotations. More \
+  on that in the :doc:`parameter description section</parameter_description>` of this guide.
+
+
+We can then see in the OMERO activities that the Key-Value pairs were added to 5 \
+datasets out of the 11 present in this project (as expected). We also see three set of \
+key-value pairs, one for each namespace we used in the csv.
+
+.. figure:: images/annotation_07_KV_import_printout.png
+
+  *The script output (5 entries in the csv matched to 5 dataset out of 11) and the\
+  resulting key-value pairs annotation.*
+..
+
+Converting the Key-Value pairs namespace
+----------------------------------------
+
+Key-Value pairs are assigned a category/label (known as namespace). \
+This grants flexibility so that different annotations can be \
+distinguished or isolated (like for exporting/deleting only those with a given \
+namespace).
+
+   In fact, if you created Key-Value pairs in OMERO.web, you have used \
+   namespaces without noticing it: OMERO assigns by default the \
+   "client namespace" (``openmicroscopy.org/omero/client/mapAnnotation`` in full)\
+   , a special namespace recognized by OMERO.web allowing to edit them.
+
+We would like to do a few changes in OMERO.web, and for that we would need to \
+change the namespace to the default OMERO.web namespace. We indicate the namespace we \
+want to edit and leave the second field empty (default namespace is then used).
```

Comment:
Maybe that is why you are converting the namespaces in frist place, to get to the situation where you can edit them later in webclient, because they are in the ``right`` ns now ?
Please try to motivate the user better in terms of why should they use your valuable script ? Ta